.github/workflows/ruby.yml

```diff
@@ -40,10 +40,10 @@ jobs:
         run:  bundle exec rake compile
 
       - name: test
-        timeout-minutes: 10
+        timeout-minutes: 8
         run:  |
           # RubyGems < 3, no frozen strings
-          if [[ "2.6" < "${{ matrix.ruby }}.0" ]]; then
+          if [[ "2.6" < "${{ matrix.ruby }}.0" ]] && [[ "${{ matrix.ruby }}" != "truffleruby" ]]; then
```

Comment:
truffleruby not needed here, right?

Reply:
Removed here and line 146, as truffleruby-head is ok